.pipelines/windows-image.yaml

```diff
@@ -0,0 +1,55 @@
+stages:
+- stage: build_windows_images
+  displayName: Build Windows Images
+  dependsOn:
+    - "setup"
+  jobs:
+    - job: NPM
+      displayName: Build NPM
+      variables:
+          tag: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.Tag'] ]
+      pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT_WINDOWS)"
+      steps:
+        - powershell: |
+            powershell.exe -command "& { . .\windows.ps1; azure-npm-image $(tag)-windows-amd64 }"
+          name: "build_npm"
+          displayName: "Build"
+
+        - task: Docker@2
+          displayName: Docker Login
+          inputs:
+            containerRegistry: $(ACR_SERVICE_CONNECTION)
+            command: 'login'
+            addPipelineData: false
+
+        - powershell: |
+            docker tag acnpublic.azurecr.io/azure-npm:$(tag)-windows-amd64 acnpublic.azurecr.io/azure-npm:$(tag)-windows-amd64-test
```

Comment:
I agree, but I'd like to leave this as is for the moment, and take that in https://github.com/Azure/azure-container-networking/pull/1079 since that's the theme for that pr, and this will be refactored again there